frontend/javascripts/messages.tsx

```diff
@@ -45,6 +45,7 @@ export const settings: Partial<Record<keyof RecommendedConfiguration, string>> =
   gpuMemoryFactor: "Hardware Utilization",
   overwriteMode: "Volume Annotation Overwrite Mode",
   useLegacyBindings: "Classic Controls",
+  renderWatermark: "Include Watermark",
```

Comment:
I'm not entirely sure about the naming/description here. I would propose "Watermark on Screenshots" to make it clear that the watermark refers to screenshots. Also, watermark might have a slightly negative connotation so we could also refer to it as "Logo on Screenshots" but I might be overthinking it :thinking: @philippotto any opinion?

For reference, this is what it currently looks like (with the description being "Show a watermark in the lower-left corner in each screenshot.")

![image](https://user-images.githubusercontent.com/1702075/232471007-48bc0ad1-3a0c-4331-af0b-dff344c41f68.png)

Reply:
Yes, I would change "watermark" to "logo", too. In the tooltip, it could even say "WEBKNOSSOS logo".

>  so we could also refer to it as "Logo on Screenshots"

Maybe "Logo **in** Screenshots"?

Reply:
I am also in favor of saying “logo” instead of “watermark” :)